Sources/Rendering/Misc/FullScreenRenderWindow/index.js

```diff
@@ -94,8 +94,14 @@ function vtkFullScreenRenderWindow(publicAPI, model) {
   model.interactor.initialize();
   model.interactor.bindEvents(model.container);
 
-  // Expose background
-  publicAPI.setBackground = model.renderer.setBackground;
+  publicAPI.setBackground = macro.chain(
+    publicAPI.setBackground,
+    model.renderer.setBackground
+  );
+  publicAPI.getBackground = model.renderer.getBackground;
+
+  // Update BG color
+  publicAPI.setBackground(...model.background);
```

Comment:
It is not needed anymore (since if the `background` is given in the initialValues it wall the setters, or if it is not as it is in the defaultValues it will be pushed in the initialValues)